From ca9314ae063f5c8ca7b636c4a90f6425ec084c16 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Tue, 1 Oct 2024 17:01:41 -0700
Subject: [PATCH 6/6] disable cuda specific gds build

gds build causes build error if it is build on rocm sdk
environment which does not have cuda home specified.

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 build_rocm.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build_rocm.sh b/build_rocm.sh
index 6cbf9e0e..c33076e0 100755
--- a/build_rocm.sh
+++ b/build_rocm.sh
@@ -19,6 +19,6 @@ mkdir -p deepspeed/ops/spatial
 export DS_ACCELERATOR=cuda
 # install command will create wheel and install it. bdist_wheel comamnd would only create the wheel
 # -j"${BUILD_CPU_COUNT_SAFE}" causes build breaks thats gets fixed by restating build. Using -j1 for now
-AMDGPU_TARGETS="${amd_target_gpu}" DS_BUILD_AIO=0 DS_BUILD_FP_QUANTIZER=0 DS_BUILD_QUANTIZER=0 DS_BUILD_SPARSE_ATTN=0 DS_BUILD_RAGGED_DEVICE_OPS=0 DS_BUILD_CUTLASS_OPS=0 DS_BUILD_EVOFORMER_ATTN=0 DS_BUILD_OPS=1 python setup.py build_ext -j1 bdist_wheel
+AMDGPU_TARGETS="${amd_target_gpu}" DS_BUILD_GDS=0 DS_BUILD_AIO=0 DS_BUILD_FP_QUANTIZER=0 DS_BUILD_QUANTIZER=0 DS_BUILD_SPARSE_ATTN=0 DS_BUILD_RAGGED_DEVICE_OPS=0 DS_BUILD_CUTLASS_OPS=0 DS_BUILD_EVOFORMER_ATTN=0 DS_BUILD_OPS=1 python setup.py build_ext -j1 bdist_wheel
 
 #DS_BUILD_UTILS=1 DS_BUILD_CPU_ADAGRAD=1 DS_BUILD_RANDOM_LTD=1 DS_BUILD_CPU_ADAM=1 DS_BUILD_FUSED_ADAM=1 DS_BUILD_FUSED_LAMB=1 DS_BUILD_CCL_COMM=1 python setup.py develop
-- 
2.34.1

